packages/typescript/ai-mistral/src/utils/client.ts

```
@@ -0,0 +1,78 @@
+import { Mistral, HTTPClient } from '@mistralai/mistralai'
```

ESLint

[error] Member 'HTTPClient' of the import declaration should be sorted alphabetically.

(sort-imports)